contents/support.md

```diff
@@ -8,28 +8,22 @@ Whether or not you pay to use PostHog, you can always find help at:
 
 ## Documentation
 
-We have written out extensive [docs](/docs) for how to deploy PostHog.
-
-There is also the PostHog [FAQ](/faq).
-
-## Issues or feature requests
-
-If you've found a bug? [Raise an issue](https://github.com/PostHog/posthog/issues).
-
-Want to suggest new functionality? [Raise an issue](https://github.com/PostHog/posthog/issues) too.
+We have extensive [docs](/docs) on how to deploy and use PostHog, which are updated almost every day. You can also find answers to more general questions in the PostHog [FAQ](/faq).
 
 ## Community Slack
 
-For urgent isses, we have a [community Slack](/slack) group.
+For general product support, join our [community Slack](/slack) group - 1,800 members and counting! It is by far the quickest way to get help from the PostHog team and wider community. 
+
+## Bugs or feature requests
 
-## Email
+Found a bug? [Raise an issue](https://github.com/PostHog/posthog/issues)!
 
-Email [hey@posthog.com](mailto:hey@posthog.com) with any miscellaneous questions that don't fit into the above.
+Want to suggest new functionality? [Raise an issue](https://github.com/PostHog/posthog/issues) too.
 
 ## Sales enquiries
 
-Interested in PostHog's team providing support? [Contact us](https://share.hsforms.com/1-IVCY9gNRvaZBajMt_UPIg4559u) for help managing your deployment.
+Interested in PostHog's team providing support? [Contact us](https://share.hsforms.com/1-IVCY9gNRvaZBajMt_UPIg4559u) for help managing your deployment. We offer extensive [paid support options](/docs/user-guides/support).
```

Comment:
@camerondeleone could you provide a relevant calendly link here? You can use this to discuss support with folk needing it then to send them to the right place.